src/frontend/komponenter/Fagsak/Søknad/MålformVelger.tsx

```diff
@@ -0,0 +1,53 @@
+import * as React from 'react';
+import { Undertittel } from 'nav-frontend-typografi';
+import PanelBase from 'nav-frontend-paneler';
+import { useBehandling } from '../../../context/BehandlingContext';
+import { ISøknadDTO, Målform, målform } from '../../../typer/søknad';
+import { Radio } from 'nav-frontend-skjema';
+import { FamilieRadioGruppe } from '@navikt/familie-form-elements/dist';
+
+interface IProps {
+    settSøknadOgValider: (søknad: ISøknadDTO) => void;
+    søknad: ISøknadDTO;
+}
+
+const MålformVelger: React.FunctionComponent<IProps> = ({ settSøknadOgValider, søknad }) => {
+    const { erLesevisning } = useBehandling();
+    const lesevisning = erLesevisning();
+
+    const radioOnChange = (målform: Målform) => {
+        settSøknadOgValider({
+            ...søknad,
+            søkerMedOpplysninger: {
+                ...søknad.søkerMedOpplysninger,
+                målform: målform,
+            },
+        });
+    };
+
+    return (
+        <PanelBase key={'målform'}>
+            <Undertittel children={'Målform'} />
+            <br />
+            <FamilieRadioGruppe
+                erLesevisning={lesevisning}
+                verdi={søknad.søkerMedOpplysninger.målform}
```

Comment:
Har ikke denne en legend prop man heller burde bruke? Eventuelt kan Undertittel over være `<Label htmlFor={familieRadioGruppe} />`.

Reply:
Litt usikker egentlig, for i skissene ser det ut som at det er en undertittel. Veit ikke hvordan dere pleier å gjøre det? Er brukt undertittel på de andre i allefall.

![Screenshot 2020-09-08 at 10 39 28](https://user-images.githubusercontent.com/8656966/92453997-4a40d880-f1c0-11ea-9f40-54e4387a5389.png)